packages-exp/auth-types-exp/index.d.ts

```diff
@@ -411,3 +411,18 @@ export interface UserMetadata {
   readonly creationTime?: string;
   readonly lastSignInTime?: string;
 }
+
+/**
+ * Additional user information.
+ */
+export interface AdditionalUserInfo {
+  readonly isNewUser: boolean;
+  readonly profile?: UserProfile;
+  readonly providerId: ProviderId | null;
```

Comment:
providerId can't be null https://firebase.google.com/docs/reference/js/firebase.auth#additionaluserinfo

Reply:
Well that just isn't true:

https://github.com/firebase/firebase-js-sdk/blob/799507101171bce76879d93ee135b12fb922ffcf/packages/auth/src/additionaluserinfo.js#L137

Reply:
huh, i wonder if our docs need to be updated then